packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -52,17 +53,16 @@ export class MultiSelectExample extends BaseExample<IMultiSelectExampleState> {
         });
 
         const initialContent = this.state.hasInitialContent ? (
-            <MenuItem disabled={true} text={`${TOP_100_FILMS.length} items loaded.`} />
+            <MenuItem disabled={true} text={`${Films.items.length} items loaded.`} />
         ) : (
             undefined
         );
 
         return (
             <FilmMultiSelect
+                {...Films}
```

Comment:
this is pretty confusing / too clever. please refactor to export something like `multiSelectProps` from the "./films" module

Reply:
then you also wouldn't have awkward looking code like `type Film = Films.Film;`